src/dialogs/mqtt-device-debug-info-dialog/mqtt-payload.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  LitElement,
+  html,
+  TemplateResult,
+  customElement,
+  property,
+} from "lit-element";
+import { safeDump } from "js-yaml";
+
+@customElement("mqtt-payload")
+class MQTTPayload extends LitElement {
+  @property() public payload?: object | string;
+  @property() public _showAsYaml: boolean = false;
+  @property() public _showDeserialized: boolean = false;
```

Comment:
```suggestion
  @property() public showAsYaml: boolean = false;
  @property() public showDeserialized: boolean = false;
```

Only private variable should be prepended with `_`